emojiHeatmapper/index.html

```diff
@@ -57,6 +59,7 @@
   <!-- jQuery JS -->
   <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
   <script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
+  <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
   <!-- Bootstrap JS -->
```

Comment:
why so many versions of jquery, and why are you not using minified version for the last inclusion?